app/invocation/cache_requests_card.tsx

```diff
@@ -302,7 +302,7 @@ export default class CacheRequestsCardComponent extends React.Component<CacheReq
           {/* Filtering controls */}
           <div className="separator" />
           <label>Show</label>
-          <Select value={this.getFilterIndex()} onChange={this.onChangeFilter.bind(this)}>
+          <Select id="filter-cache-requests" value={this.getFilterIndex()} onChange={this.onChangeFilter.bind(this)}>
```

Comment:
nit: we usually use `className` instead of `id`

Reply:
Trying webdriver tests to classNames can be brittle (i.e. you could accidentally break a webdriver test by changing the style of an element).

A few options (I'm open to any of theses):
- We could have a special classname prefix that we know will be selected by webdriver, so we shouldn't change it
- We could use another attribute that's only used for webdriver ids like `debug-id="foo"`
- We could make IDs be our 
- We could roll with class names, and just be careful to search the webdriver tests before changing them

Here's a post on selenium locator best practices: https://www.screenster.io/selenium-locators-best-practices-7-helpful-tips-for-ui-testers/

Reply:
I like the `debug-id` idea; we might have to call it `data-debug-id` to make React/TypeScript happy.

Reply:
Good idea. Modified to use `debug-id`